make/devices.cmake

```diff
@@ -363,10 +403,12 @@ function(DEFINE_DEVICE)
         ${QUIET_CMD} ${VPR} ${DEVICE_MERGED_FILE}
         --device ${DEVICE_FULL}
         ${symbiflow-arch-defs_SOURCE_DIR}/common/wire.eblif
-        --route_chan_width 100
+        --route_chan_width 6
```

Comment:
Why 6?

Reply:
It's even and small.  Emitting, and filtering a virtual RR graph with channel width 100 is much much slow than a 6 width channel.  The virtual RR graph is pretty much only used for seeding the real RR graph.